src/tools/rbd/action/Import.cc

```diff
@@ -526,7 +526,7 @@ int execute_diff(const po::variables_map &vm,
 }
 
 Shell::Action action_diff(
-  {"import-diff"}, {}, "Import an incremental diff.", "", &get_arguments_diff,
+  {"import-diff"}, {}, "Apply an incremental diff to current image content. (will change the image content)", "", &get_arguments_diff,
```

Comment:
I'm not sure if "current image content" is well defined.  I would suggest the following:
```suggestion
  {"import-diff"}, {}, "Apply an incremental diff to image HEAD and create a snapshot.", "", &get_arguments_diff,
```

Reply:
@idryomov Thanks for your comment. Changed to 
`Apply an incremental diff to image HEAD, then create a snapshot.`